oeps/oep-0037/decisions/0001-yaml-centralization.rst

```diff
@@ -0,0 +1,34 @@
+Centralization of Development Data Yaml files
+=============================================
+
+Status
+------
+
+Draft
```

Comment:
Any reason this one is Draft and the other two are Accepted?

Reply:
This should be Accepted.

Reply:
Will change in future commit